src/App.spec.ts

```diff
@@ -55,6 +55,34 @@ describe('App', () => {
         assert.instanceOf(app, App);
       });
     });
+
+    describe('with unsuccessful single team authorization results', () => {
+      it.only('should throws', async () => {
```

Comment:
```suggestion
      it('should throws', async () => {
```

Reply:
Also, please apply `prettier -w src/` before committing your changes.

Reply:
oh sorry ! I forgot it ! ^^'